client/gatsby-node.js

```diff
@@ -77,6 +77,7 @@ exports.createPages = function createPages({ graphql, actions, reporter }) {
                   block
                   certification
                   challengeType
+                  dashedName
```

Comment:
Can we simplify further by using just `block`? Having both seems redundant and `block` is more descriptive.

Reply:
It's a little confusing, but this is the challenge title `dashedName` - not the block. I added it because of the change in `completion-modal.tsx` - it was using the `dasherize` function to  dasherize the challenge title as a download filename. So now it just uses the `dashedName` we give the challenge.

Reply:
Ooops, I didn't see that. Thanks for explaining!